include/ignition/rendering/ArrowVisual.hh

```diff
@@ -41,6 +41,9 @@ namespace ignition
       /// \brief Get arrow-shaft visual
       /// \return The arrow-shaft visual
       public: virtual VisualPtr Shaft() const = 0;
+
+      /// \brief set true to show the arrow head, false otherwise
+      public: virtual void ShowArrowHead(bool b) = 0;
```

Comment:
style nitpick: can you add `_` prefix to arg name, i.e.` bool _b`

and add doxygen comment for  `_b`